### PR TITLE
EZEE-3455: [Image upload] no validation message for files bigger than 2mb

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/EZEE-3455
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/fieldType/ezimage.js [hunk 1]
```diff
@@ -48,15 +49,14 @@
...
-            const container = input.closest('.ez-field-edit--ezimage');
-            const method = !!container.querySelector(`.${this.classInvalid}`) ? 'add' : 'remove';
-
-            container.classList[method](this.classInvalid);
+            return {
+                isError: false,
+            };
         }
-
+
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/fieldType/ezimage.js:59]:
```suggestion

```

File: src/bundle/Resources/public/js/scripts/fieldType/ezimage.js [hunk 2]
```diff
@@ -80,7 +80,7 @@
         }
     }
 
-    [...document.querySelectorAll(SELECTOR_FIELD)].forEach((fieldContainer) => {
+    [...document.querySelectorAll(SELECTOR_FIELD) ].forEach((fieldContainer) => {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/fieldType/ezimage.js:83]:
```suggestion
    [...document.querySelectorAll(SELECTOR_FIELD)].forEach((fieldContainer) => {
```

**Review by dew326 — CHANGES_REQUESTED** [review]

**Review by dew326 — APPROVED** [review]

**Review by bogusez — APPROVED** [review]

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by lucasOsti** [comment]
> Could you merge it up?

Done

**Pull request merged**